ocs_ci/deployment/deployment.py

```diff
@@ -1090,6 +1090,11 @@ def setup_local_storage(storageclass):
             constants.LOCAL_VOLUME_SET_YAML
         )
 
+        # as a workaround we are setting Rotational to deviceMechanicalProperties to detect HDD disk
```

Comment:
do we have an issue to track the W/A?
also, do we need to set this value everytime?

Reply:
i have created  an issue here #3098, yes need to set that value of all the deployment of VMware LSO